src/profile-logic/cpu.js

```diff
@@ -191,124 +235,3 @@ function findClosestNonNullValueToIdx(
 
   return findClosestNonNullValueToIdx(array, idx, ++distance);
 }
-
-// Threshold to be used for checking if a single threadCPUDelta value is above
-// the maxThreadCPUDelta. If the sample CPU value is using more than 10% of CPU,
-// it's safe to assume that this sample is active and non-idle.
-const CPU_IDLENESS_PERCENTAGE = 0.1;
-
-/**
- * Compute the activity percentages for each threads by looking at their CPU usage
- * per thread. This will be used to determine the idle threads and also to
- * visualize the activity of threads in the UI.
- * A sample is considered active if its CPU value is above
- * `maxThreadCPUDelta * CPU_IDLENESS_PERCENTAGE`.
- */
-export function computeThreadActivityPercentages(
```

Comment:
It's nice that we are actually getting rid of this code. I wasn't very satisfied with this when I was writing 😄 Also when I was writing this, I was thinking about displaying the CPU usage percentages somewhere in the UI. But probably we can make something similar with sampleScore.

Reply:
Ah, I see, thanks for the background. Yes, I think displaying the sample score in the UI would be a good idea. Maybe in the all-thread dropdown, we could show some kind of bar graph or shaded highlighting for each thread, so that it becomes clearer which threads are worth showing.